catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/taxonomy/IsrAttributes.java

```diff
@@ -0,0 +1,367 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.catalog.core.api.impl.taxonomy;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.codice.alliance.catalog.core.api.taxonomy.Isr;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.taxonomy.MetacardComponent;
+
+/**
+ * This class provides attributes that relate to intelligence, search, and reconnaissance
+ * <p>
+ * https://codice.atlassian.net/wiki/display/DDF/Attributes
+ */
+public class IsrAttributes implements Isr, MetacardComponent {
+
+    private static final Set<AttributeDescriptor> DESCRIPTORS = new HashSet<>();
+
+    private static final String NAME = "isr";
+
+    static {
+        DESCRIPTORS.add(new AttributeDescriptorImpl(CLOUD_COVER,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.INTEGER_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(VIDEO_MOVING_TARGET_INDICATOR_PROCESSED,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.BOOLEAN_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(COMMENTS,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
```

Comment:
Shouldn't string types be tokenized? Not sure if we're concerned about index bloat or something here, but I would think the answer is yes.